Source/Diagnostics/ComputeDiagFunctors/ParticleReductionFunctor.H

```diff
@@ -41,10 +45,15 @@ public:
 private:
     int const m_lev; /**< level on which mf_src is defined */
     int const m_ispec; /**< index of species to average over */
+    bool const m_do_filter; /**< whether to apply #m_filter_fn */
```

Comment:
Is the `#` intentional?
```suggestion
    bool const m_do_filter; /**< whether to apply m_filter_fn */
```

Reply:
The `#` in the doxygen comment links to the member variable, e.g. 
https://warpx.readthedocs.io/en/latest/_static/doxyhtml/class_particle_reduction_functor.html#af0e36574686a5b6163543f31a2ff43b5

It's definitely not necessary, though.

Reply:
Oh cool, I thought it magically maps them by name. TIL! :)